.travis.yml

```
@@ -117,14 +117,17 @@ jobs:
       - ./bin/build-helm
       - mkdir helm-charts
       - cp helm/quarks-job*.tgz helm-charts/
+      - git tag -a $ARTIFACT_VERSION -m "tag $ARTIFACT_VERSION"
       deploy:
-        provider: s3
-        access_key_id: "${AWS_ACCESS_KEY}"
-        secret_access_key: "${AWS_SECRET_KEY}"
-        bucket: "${S3_BUCKET}"
-        acl: public_read
+      - provider: script
+        script: git clone https://CFContainerizationBot:$GITHUB_TOKEN@github.com/cloudfoundry-incubator/quarks-helm.git ./updated/ && cp helm/quarks-job*.tgz updated/ && ./bin/publish-helm-repo
```

I would have expected this folder:
```suggestion
        script: git clone https://CFContainerizationBot:$GITHUB_TOKEN@github.com/cloudfoundry-incubator/quarks-helm.git ./updated/ && cp helm-charts/quarks-job*.tgz updated/ && ./bin/publish-helm-repo
```